### ISSUE TITLE
SDash created files and commits are now getting committed under svengato?

### ISSUE THREAD

**Issue by sdash-github** [PeanutBase/jekyll-peanutbase#37]
1.  I just committed a bunch of files "Expression related tool and pages added.
62d844f" and the individual files in github says svengato committed them. Although, I created them and committed them???
2.  In pb-jekyll why there is now a permission issue? I have to do sudo to git add, etc.  This could be related to (1.) above.

**Comment by svengato** [comment]
I removed my global Git configuration settings and replaced them with local ones (that should affect only my commits). This seems to work now.

**Comment by sdash-github** [comment]
Andrew committed a bunch of files on 06-12-2023 and they still show up under svengato:
  ba9d... to c60b... 
I am going to test a few more of mine today and see what happens.

**Comment by sdash-github** [comment]
c60b5b1..ef3a987  committed by sdash-github shows up under svengato.
This is inspite of svengato "removed my global Git configuration settings and replaced them with local ones".
Not an urgent issue now and we can discuss it in our Thu meeting.

**Comment by sdash-github** [comment]
I think the culprit is the .git/config file entry. I just don't know yet what it should be instead. May be another server where things work properly should help to modify these lines to correct the problem.

```
@dev-pb-jekyll /var/www/jekyll-peanutbase]
<$> less -N .git/config
...  ...  ...
     21 [user]
     22         name = Sven Redsun
     23         email = svengato@svengato.com
```

**Comment by adf-ncgr** [comment]
Yes, I was just looking at that. I'd try just removing them and see what happens.

**Comment by sdash-github** [comment]
Solved: Now my commit shows up as mine after commenting out the entire user lines per below in the .git/config file.
```
     21 #[user]
     22 #       name = Sven Redsun
     23 #       email = svengato@svengato.com
```
Observe a few more before closing issue.
